components/user_settings/user_settings_advanced.jsx

```diff
@@ -447,76 +451,78 @@ export default class AdvancedSettingsDisplay extends React.Component {
 
         let previewFeaturesSection;
         let previewFeaturesSectionDivider;
-        if (this.state.preReleaseFeaturesKeys.length > 0) {
-            previewFeaturesSectionDivider = (
-                <div className='divider-light'/>
-            );
+        if (this.state.previewFeaturesEnabled) {
```

Comment:
Could we combine this if statement with the one below? Gets rid of a bit of unneeded nesting and makes way easier to read

Reply:
@jwilander rebased to latest and combined ifs to remove unneeded nesting.